.github/workflows/release.yaml

```diff
@@ -52,3 +57,27 @@ jobs:
             -H "Accept: application/vnd.github.v3+json" \
             https://api.github.com/repos/${{github.repository}}/actions/workflows/${{env.QUILL_ICONS_PARK_WORK_FLOW_FILE}}/dispatches \
             -d '{"ref":"main"}'
+
+  send_slack_notification:
+    name: Send Release Slack notification
+    runs-on: ubuntu-latest
+    if: always()
+    needs: [release]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - name: Conclusion
+        uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5
```

Comment:
`technote-space/workflow-conclusion-action@v3.0.3` ?

Reply:
As @jim-deriv mentioned this is related to security team's recommendation